tools/serverpod_cli/bin/serverpod_cli.dart

```diff
@@ -40,6 +40,15 @@ final runModes = <String>['development', 'staging', 'production'];
 final Analytics _analytics = Analytics();
 
 void main(List<String> args) async {
+  if (Platform.isWindows) {
```

Comment:
I'm thinking that perhaps it would be more clean to encapsulate the platform details in the initializeLogger method. I.e.:

```dart
initializeLogger();
```

Reply:
Nothing obstructing us from doing that.

I wanted to keep the `initialiseLogger` method open so that we potentially could pass in a custom logger,  from the test for example. A good middle ground could be to take a logger as an optional argument and then use this initialisation setup if no logger is provided and otherwise use the provided one.